app/screens/channel_members/index.js

```diff
@@ -13,10 +13,11 @@ import {getProfilesInChannel, searchProfiles} from 'mattermost-redux/actions/use
 import ChannelMembers from './channel_members';
 
 function mapStateToProps(state) {
+    const currentChannelMembers = getProfilesInCurrentChannel(state).filter((member) => member.delete_at === 0);
```

Comment:
@saturninoabril you have the selector factory for `makeGetProfilesInChannel` and you can pass an argument for `skipInactive` wouldn't that work better?